airflow/auth/managers/base_auth_manager.py

```diff
@@ -102,3 +154,8 @@ def security_manager(self, security_manager: AirflowSecurityManager):
         :param security_manager: the security manager
         """
         self._security_manager = security_manager
+
+    @staticmethod
+    def is_dag_resource(resource_type: ResourceType) -> bool:
+        """Determines if a resource relates to a DAG."""
+        return resource_type == ResourceType.DAG
```

Comment:
What’s the use for this method?

Reply:
I'll remove this one, I am currently working on refactoring this PR

Reply:
Putting it as draft for now

Reply:
Done :). Feel free to take a look if you're interested :)